functions/view/advertiser/demographics-survey.hbs

```diff
@@ -0,0 +1,179 @@
+                type='radio'
+                id='africa'
+                name='continent'
+                value='africa'
+                checked
+              />&nbsp Africa</label>
+            <label class='demo__radio-label' for='antarctica'><input
+                type='radio'
+                id='antarctica'
+                name='continent'
+                value='antarctica'
+              />&nbsp Antarctica</label>
+            <label class='demo__radio-label' for='asia'><input
+                type='radio'
+                id='asia'
+                name='continent'
+                value='asia'
+              />&nbsp Asia</label>
+            <label class='demo__radio-label' for='australia'><input
+                type='radio'
+                id='australia'
+                name='continent'
+                value='australia'
+              />&nbsp Australia</label>
+            <label class='demo__radio-label' for='europe'><input
+                type='radio'
+                id='europe'
+                name='continent'
+                value='europe'
+              />&nbsp Europe</label>
+            <label class='demo__radio-label' for='north-america'><input
+                type='radio'
+                id='north-america'
+                name='continent'
+                value='north-america'
+              />&nbsp North America</label>
+            <label class='demo__radio-label' for='south-america'><input
+                type='radio'
+                id='south-america'
+                name='continent'
+                value='south-america'
+              />&nbsp South America</label>
+          </div>
+          <br />
+          <div>
+            <button class='demo__submit-button' type='submit'>Submit</button>
+            <button class='demo__continue-button'>
+              <a href='{{{publisherAUrl}}}/demographics-measurement'>
+                Continue
+              </a>
+            </button>
+          </div>
+        </form>
+      </card>
+      <card class='demo__description'>
+        <h4>Description</h4>
+        <p>
+          You may want to measure the demographics of the users who has seen some content you have embedded across
+          different sites.
+        </p>
+        <p>
+          For this demo, visit the survey page to enter your demographic data where the information you enter is added
+          to Shared Storage. Then visit either Publisher A or B sites which will trigger the aggregatable report to be
+          sent once across both sites. The content ID, age group ID, and the geography ID dimensions are encoded into
+          the aggregation key (bucket), and the count is used as the aggregatable value.
+        </p>
+        <p>
+          The summary report generated will provide information such as "Approximately 391 users who have seen the
+          content ID 743 are between the age of 18-25 and are from New York City."
```

Comment:
maybe (here and elsewhere): s/New York City/Europe/ (or some other continent) for consistency